kube/client.go

```diff
@@ -0,0 +1,105 @@
+package kube
+
+import (
+	"errors"
+	"log"
+	"os"
+
+	"github.com/generationtux/brizo/config"
+	"github.com/mitchellh/go-homedir"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// Client builds the kubernetes client
+func Client() (*kubernetes.Clientset, error) {
+	if config.Kubernetes.External {
+		return externalClient()
+	}
+
+	return internalClient()
+}
+
+// internalClient builds the kube client configured to run inside a kube cluster
+func internalClient() (*kubernetes.Clientset, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		var client *kubernetes.Clientset
```

Comment:
Is there some sort of initialization that needs to happen at this point for this `Clientset`?